roles/contiv_network/tasks/main.yml

```diff
@@ -82,8 +85,12 @@
   copy: src=netmaster.service dest=/etc/systemd/system/netmaster.service
   when: run_as == "master"
 
+- name: reload system config for netmaster
+  shell: systemctl daemon-reload
+  when: run_as == "master"
```

Comment:
OK. Could you use the daemon-reload option in service module rather than have a separate task for this?

(Ref: https://docs.ansible.com/ansible/systemd_module.html#options)